src/common.rs

```diff
@@ -517,6 +519,47 @@ pub fn generate_keypair() -> color_eyre::eyre::Result<KeyPairProperties> {
     Ok(key_pair_properties)
 }
 
+fn compute_hash(bytes: &[u8]) -> [u8; 32] {
+    let mut hasher = Sha256::new();
+    hasher.update(&bytes);
+
+    let result = hasher.finalize();
+    let mut hash = [0u8; 32];
+    hash.copy_from_slice(&result[..]);
+
+    hash
+}
+
+pub fn print_full_signed_transaction(transaction: near_primitives::transaction::SignedTransaction) {
+    eprintln!("{:<25} {}\n", "signature:", transaction.signature);
+    crate::common::print_full_unsigned_transaction(transaction.transaction);
+}
+
+pub fn print_full_unsigned_transaction(transaction: near_primitives::transaction::Transaction) {
+    let bytes = transaction
+        .try_to_vec()
+        .expect("Transaction is not expected to fail on serialization");
+    eprintln!("transaction payload (without signature) hash:");
+    eprintln!(
+        "{:<25} {}",
+        "SHA256 hash(hex):",
```

Comment:
In NEAR, transaction hash is usually represented as base58 and even though I wish we wouldn't reinvent the wheel and used hex since day one, but we did not. Unless Ledger requires and shows hex, let's use it, otherwise, I would make things simpler and only use base58 and call it "Transaction hash (Base58-encoded SHA-256 hash)".